### PR TITLE
Container.py fix to allow get_details

### PR DESCRIPTION
**Pull request by pavelzag**

### CODE REVIEW

File: cfme/common/provider.py [hunk 1]
```diff
@@ -75,8 +75,8 @@ def form(self):
                     ('default_principal', Input("default_userid")),
                     ('default_secret', Input("default_password")),
                     ('default_verify_secret', Input("default_verify")),
-                    ('token_secret', Input('bearer_password')),
```
Comment (psav) [inline cfme/common/provider.py]: Is this changed in both 5.5 and 5.6?

**Comment by dajoRH** [comment]
Lint report for commit aa972a8fa772c59bdc91ac1fc3b220f0baace5b6:

`cfme/containers/container.py`:
- :heavy_exclamation_mark: Line 62:51: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 314bf8fe865d52680d70a1cb53ee0887066e9246:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/common/provider.py (hunk 1 above)
Comment (psav) [inline cfme/common/provider.py]: I believe this is addressed in another PR

**Comment by psav** [comment]
Happy to merge if we remove the unrelated or at least, "WIP in another PR" version string change

File: cfme/common/provider.py (hunk 1 above)
Comment (pavelzag) [inline cfme/common/provider.py]: That's what changed from 5.5 to 5.6
The provider validation fails without it

**Comment by pavelzag** [comment]
Can you elaborate on this Pete? Not sure what should I do here

File: cfme/containers/container.py [hunk 2]
```diff
@@ -44,8 +45,18 @@ def navigate(self, detail=True):
                     'containers_container_detail', context={
                         'container': self, 'pod': self.pod})
         else:
-            sel.force_navigate('containers_container', context={'container': self})
+            sel.force_navigate('containers_container',
+                               context={'container': self, 'pod': self.pod})
```
Comment (mfalesni) [inline cfme/containers/container.py]: Think about the nav tree logic. If container provides a pod object, then it is pointless to pass the pod along.

The nav tree pulls the pod from container, line 16 and 20 ;)

File: cfme/containers/container.py [hunk 3]
```diff
@@ -21,7 +22,7 @@
 )
 
 
-class Container(Taggable, SummaryMixin):
+class Container(Taggable):
```
Comment (mfalesni) [inline cfme/containers/container.py]: Why are you removing the SummaryMixin?

Comment (pavelzag) [inline cfme/containers/container.py]: That's weird. I did not want to do that..

Comment (pavelzag) [inline cfme/containers/container.py]: I see I did some mess there..
Can we revert all this changes and I'll try to add them again the way I should've?

Comment (mfalesni) [inline cfme/containers/container.py]: You can just put the things back in in the next commit ...

**Comment by dajoRH** [comment]
Lint report for commit dc90b2b971d901233b1a884443514fc5f3e44d29:

`cfme/containers/container.py`:
- :heavy_exclamation_mark: Line 63:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 63:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit df51b009f1b2caf0807b610967dff187ebeda91e:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request closed without merging**